release/scripts/generate_notice.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
/bin/bash? where is it usually found? across mac and linux?

Reply:
Made it /bin/bash for consistency in all release scripts